src/lib.rs

```diff
@@ -45,7 +45,7 @@ pub struct Baid58<const LEN: usize> {
 impl<const LEN: usize> Baid58<LEN> {
     /// # Panics
     ///
-    /// If HRI static string is longer than
+    /// If HRI static string is longer than [HRI_MAX_LEN]
     pub fn with(hri: &'static str, payload: [u8; LEN]) -> Self {
         debug_assert!(hri.len() <= HRI_MAX_LEN, "HRI is too long");
```

Comment:
why we disable this check in release build by using `debug_assert`?

Reply:
Since the assumption that the bug will be detected during the debugging. But you are right it is better to have just an assert